src/emcpy/plots/create_plots.py

```diff
@@ -440,9 +440,22 @@ def _map_gridded(self, plotobj, ax):
                     'markersize', 'colorbar']
         inputs = self._get_inputs_dict(skipvars, plotobj)
 
-        cs = ax.pcolormesh(plotobj.latitude, plotobj.longitude,
-                           plotobj.data, **inputs,
-                           transform=self.projection.transform)
+        # Check for 3d data
+        if plotobj.longitude.ndim == 3:
+            # Get total number of layers
```

Comment:
so the first dimension is tile? also change 'layers' to 'tiles'?

Reply:
Might add a comment to say it assumes that the Nth dimension is tile, etc.

Reply:
We need to set a standard now so would you rather (tile, x, y) or (x, y, tile)?

Reply:
IMO, `tile` should be the outermost dimension, so that if it is a singleton (ntiles=1), a `squeeze` will return `x, y` instead of `x, y, 1`

Reply:
think of it similar to the `time` dimension.